test/create.test.js

```diff
@@ -33,66 +34,95 @@ test('create', async (t) => {
-      t.true(ref.isCloakedMsgId(id1), 'has id')
+test('create more', (t) => {
+  const server = Testbot()
+
+  // this is more of an integration test over the api
+  server.tribes2.create({}, (err, data) => {
+    if (err) throw err
+
+    const { id, secret, root } = data
+    t.true(ref.isCloakedMsg(id), 'returns group identifier - groupId')
+    t.true(
+      Buffer.isBuffer(secret) && secret.length === 32,
+      'returns group symmetric key - groupKey'
+    )
+    //TODO: can we get the root msg unencrypted?
+    //t.match(
+    //  groupInitMsg.value.content,
+    //  /^[a-zA-Z0-9/+]+=*\.box2$/,
+    //  'encrypted init msg'
+    //)
+
+    server.db.get(root, (err, value) => {
+      if (err) throw err
+
+      t.deepEqual(
+        value.content,
+        {
+          type: 'group/init',
+          tangles: {
+            group: { root: null, previous: null },
+          },
+        },
+        'can decrypt group/init'
+      )
 
+      // check I published a group/add-member to myself
       pull(
-        ssb.tribes2.list(),
-        pull.collect(async (err, groups1) => {
-          if (err) t.error(err)
-
-          t.equal(groups1.length, 1, 'lists the 1 group')
-
-          const { id: id2 } = await ssb.tribes2.create().catch(t.error)
-
-          pull(
-            ssb.tribes2.list(),
-            pull.collect(async (err, groups2) => {
-              if (err) return t.error(err)
-
-              t.equal(groups2.length, 2)
-              t.equal(groups2[0].id, id1)
-              t.equal(groups2[0].secret, secret1)
-              t.equal(groups2[1].id, id2)
-
-              ssb.close(true, t.end)
-            })
+        server.db.query(where(author(server.id)), descending(), toPullStream()),
+        pull.map((msg) => msg.value.content),
+        pull.collect((err, msgContents) => {
+          if (err) throw err
+
+          t.deepEqual(
+            msgContents[0], // contents of the latest message
+            {
+              type: 'group/add-member',
+              version: 'v1',
+              groupKey: secret.toString('base64'),
+              root: root,
+              recps: [id, server.id], // me being added to the group
+              tangles: {
+                members: {
+                  root: root,
+                  previous: [root],
+                },
+                group: {
+                  root: root,
+                  previous: [root],
+                },
+              },
+            },
+            'The admin was was also added to the group'
           )
+          server.close()
+          t.end()
         })
       )
     })
-    .catch(t.error)
+  })
+})
+
+//TODO
+test.skip('tribes.create (opts.addPOBox)', (t) => {
+  const server = Server()
+
+  // this is more of an integration test over the api
+  server.tribes.create({ addPOBox: true }, (err, data) => {
+    if (err) throw err
+
+    t.true(isPoBox(data.poBoxId), 'data.poBoxId')
+
+    server.close()
+    t.end()
+  })
```

Comment:
Rather just delete this test

Reply:
same as here https://github.com/ssbc/ssb-tribes2/pull/4#discussion_r989989341